models/Todo.js

```diff
@@ -12,6 +12,14 @@ const TodoSchema = new mongoose.Schema({
   userId: {
     type: String,
     required: true
+  },
+  title: {
+    type: String,
+    required: true
+  },
+  date:{
+      type: String,
+      default: `${new Date().toDateString()} ${new Date().toTimeString()}`
```

Comment:
its for each todo created, i was doing that to create a filter by date, but i never got around to it. yeah .createdAt would make more sense. probably can scrap it though since i wont have time to put in the filter anyways. again, i forgot to change it back